### DISCUSSION TITLE
Can't add audio to a video.

### DISCUSSION THREAD

**Discussion by LaVelaPodcast** [mifi/lossless-cut#1251] [Q&A]
I'm really sorry for asking what I think is a very basic and easy question. I have tried for a couple of days, and I just can't add an audio file to a silent video and then export the 2 tracks as a complete video and audio package. I end up with the same silent video.

My process is:
1. Open my MKV file on LosslessCut
2. Extract the tracks
3. Edit audio (just compressor and normalizing) on Audacity.
4. Exporting the audio as MP3· or M4a from Audacity.
5. Open the extracted video track on LosslessCut
6. Drag and drop the mp3 or m4a file into LosslessCut
7. Check that both tracks appear as green at the Export dialogue.
8. Export as mp4

What am I doing wrong, or which step am I not taking?

Thanks in advance if someone takes the time to reply to this so basic a question.

**Comment by mifi** [comment]
Hi. what codecs are your files?
I think mp4 cannot handle mp3 audio.

**Comment by LaVelaPodcast** [comment]
Hi. Thanks for taking the time to help us. I tried both acc (m4a) and mp3,
saved from Audacity. Same result with both

El vie, 29 jul 2022 a la(s) 03:13, Mikael Finstad ***@***.***)
escribió:

> Hi. what codecs are your files?
> I think mp4 cannot handle mp3 audio.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/mifi/lossless-cut/discussions/1251#discussioncomment-3278338>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUHIPBK7X4DPOIV6LLMKSYDVWOHCNANCNFSM547COAMQ>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>


-- 

 *Enrique Herranz M.*
 +52 55 7128 8862
   (*WhatsApp only*)

**Comment by LaVelaPodcast** [comment]
This is my screen when attempting to add mp4 video and m4a audio. When I export as a final mp4, the result is only the video track, even when I selected both for export :-(

![image](https://user-images.githubusercontent.com/84838277/181845254-34603897-3d92-422c-a850-69eeb29eb906.png)

And the info from my audio is:

{
  index: 0,
  codec_name: 'aac',
  codec_long_name: 'AAC (Advanced Audio Coding)',
  profile: 'LC',
  codec_type: 'audio',
  codec_tag_string: 'mp4a',
  codec_tag: '0x6134706d',
  sample_fmt: 'fltp',
  sample_rate: '44100',
  channels: 2,
  channel_layout: 'stereo',
  bits_per_sample: 0,
  id: '0x1',
  r_frame_rate: '0/0',
  avg_frame_rate: '0/0',
  time_base: '1/44100',
  start_pts: 0,
  start_time: '0.000000',
  duration_ts: 281460300,
  duration: '6382.319728',
  bit_rate: '195770',
  nb_frames: '274864',
  extradata_size: 2,
  disposition: {
    default: 1,
    dub: 0,
    original: 0,
    comment: 0,
    lyrics: 0,
    karaoke: 0,
    forced: 0,
    hearing_impaired: 0,
    visual_impaired: 0,
    clean_effects: 0,
    attached_pic: 0,
    timed_thumbnails: 0,
    captions: 0,
    descriptions: 0,
    metadata: 0,
    dependent: 0,
    still_image: 0,
  },
  tags: {
    language: 'und',
    handler_name: 'SoundHandler',
    vendor_id: '[0][0][0][0]',
  },
}

**Reply by mifi** [reply]
That's odd. this is supposed to work. Could you try to instead open the video and then add the audio track? and try to export as MOV or MKV instead, and see if the audio works then?
